### ISSUE TITLE
@types/es6-promise breaking typescript compile

### ISSUE THREAD

**Issue by phillipb** [searchkit/searchkit#597]
I think it would be safe to release of version of 2.2 that just simply moves `@types/es6-promise` dependency to be a devDependency. It's currently causing my ts compile to fail because I'm compiling with the es6 lib. Is there anyway, someone could do that? Would hate to have to fork it for that. 

```
(11,15): Duplicate identifier 'Promise'.
```

**Comment by ssetem** [comment]
hi @phillipb  v2.3.0 has fixed this issue I think

**Comment by phillipb** [comment]
@ssetem yes, it does look like it's fixed in v2.3.0. When trying to upgrade though, I get this strange error. 

```
FastClick.js:6 Uncaught TypeError: Object prototype may only be an Object or null: undefined
    at setPrototypeOf (<anonymous>)
    at __extends (FastClick.js:6)
    at FastClick.js:15
    at Object.require.751../RenderComponent (FastClick.js:25)
    at s (_prelude.js:1)
    at _prelude.js:1
    at Object.require.757../FastClick (index.js:6)
    at s (_prelude.js:1)
    at _prelude.js:1
    at Object.require.718../AccessorManager (index.js:7)
```

**Comment by phillipb** [comment]
For posterities sake, I wasn't able to upgrade to 2.3.0 because of some issue with typescript + webpack + searchkit. Decided to stay on 2.2.0 for now and just add a `post_install` script. ```"postinstall": "rm -rf node_modules/@types/es6-promise",```
